vespalib/src/vespa/vespalib/stllike/hash_map.hpp

```diff
@@ -74,10 +74,11 @@ hash_map<K, V, H, EQ, M>::getMemoryUsed() const
              vespalib::hashtable<K, std::pair<K,V>, H, E, vespalib::Select1st<std::pair<K,V>>, M>::insert(std::pair<K,V> &&); \
     template vespalib::hashtable<K, std::pair<K,V>, H, E, vespalib::Select1st<std::pair<K,V>>, M>::insert_result \
              vespalib::hashtable<K, std::pair<K,V>, H, E, vespalib::Select1st<std::pair<K,V>>, M>::insertInternal(std::pair<K,V> &&); \
-    template class vespalib::Array<vespalib::hash_node<std::pair<K,V>>>;
```

Comment:
if you remove this line, you should also remove the continuation from the line above

Reply:
Fixed.